src/bank.rs

```diff
@@ -78,6 +72,17 @@ pub enum BankError {
 
     // Poh recorder hit the maximum tick height before leader rotation
     MaxHeightReached,
+    /// Fork is not in the Deltas DAG
+    UnknownFork,
+
+    /// The specified trunk is not in the Deltas DAG
+    InvalidTrunk,
```

Comment:
Can you replace any instance of "trunk" with "root"?  Also, that comment suggests this should be `RootNotFound`, no?

Reply:
root and trunk are different...  a trunk used more like "parent"
a root is the lowest numbered parent of all forks